linera-execution/tests/common/mod.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::thread;
+
+use linera_execution::{ExecutionRequest, QueryContext, ServiceRuntimeRequest, ServiceSyncRuntime};
+
+/// Spawns a thread running the [`ServiceSyncRuntime`] actor.
+///
+/// Returns the endpoints to communicate with the actor.
+pub fn spawn_service_runtime_actor(
```

Comment:
Could this be a method of `QueryContect` ?

Reply:
Done.